### PR TITLE
Cosmos block example

### PR DESCRIPTION
**Pull request by kruspy**

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "extends": "@graphprotocol/graph-ts/types/tsconfig.base.json",
+  "include": ["src"]
+}
```
Comment (patiee) [inline tsconfig.json:4]: new line :)

File: package.json [hunk 2]
```diff
@@ -1,26 +1,25 @@
 {
-  "name": "example",
+  "name": "cosmos-block-example",
   "version": "0.1.0",
-  "repository": "https://github.com/graphprotocol/example-subgraph",
+  "repository": "https://github.com/figment-networks/example-subgraph",
```
Comment (pienkowb) [inline package.json:4]: We should keep `graphprotocol` here.

File: package.json [hunk 3]
```diff
@@ -1,26 +1,25 @@
...
   "license": "MIT",
   "scripts": {
-    "build-contract": "solc contracts/Gravity.sol --abi -o abis --overwrite && solc contracts/Gravity.sol --bin -o bin --overwrite",
     "create": "graph create example --node https://api.thegraph.com/deploy/",
-    "create-local": "graph create example --node http://127.0.0.1:8020",
+    "create-local": "graph create cosmos-example --node http://localhost:8020",
```
Comment (pienkowb) [inline package.json:8]: `cosmos-example` => `cosmos-block-example`

Comment (pienkowb) [inline package.json:8]: Also, please move this line below `deploy` (so we have everything `-local` grouped together.

File: package.json [hunk 4]
```diff
@@ -1,26 +1,25 @@
...
     "deploy": "graph deploy example --ipfs https://api.thegraph.com/ipfs/ --node https://api.thegraph.com/deploy/",
-    "deploy-local": "graph deploy example --ipfs http://127.0.0.1:5001 --node http://127.0.0.1:8020"
+    "deploy-local": "graph deploy cosmos-example -l v0.1.0 --ipfs http://localhost:5001 --node http://localhost:8020",
+    "remove-local": "graph remove cosmos-example --node http://localhost:8020"
   },
   "devDependencies": {
-    "@graphprotocol/graph-cli": "^0.28.1",
-    "@graphprotocol/graph-ts": "^0.26.0"
+    "@graphprotocol/graph-cli": "https://github.com/figment-networks/graph-cli.git#cosmos",
+    "@graphprotocol/graph-ts": "https://github.com/figment-networks/graph-ts.git#cosmos"
```
Comment (pienkowb) [inline package.json:17]: This is good for now, but we have to remember to update this after our changes to `graph-cli` and `graph-ts` are released (a ticket?).

File: package.json [hunk 5]
```diff
@@ -1,26 +1,25 @@
...
   },
   "dependencies": {
     "babel-polyfill": "^6.26.0",
     "babel-register": "^6.26.0",
-    "truffle": "^5.0.4",
-    "truffle-contract": "^4.0.5",
-    "truffle-hdwallet-provider": "^1.0.4"
+    "pretty-error": "^4.0.0",
+    "as-proto": "^0.2.2"
```
Comment (pienkowb) [inline package.json:23]: Do we need that? `as-proto` is not used in Block Example, right?

**Review by pienkowb — COMMENTED** [review]
I've left some comments, turns out there are more than two issues 😉

**Comment by pienkowb** [comment]
BTW, in the PR we have:
> merge 5 commits into [feature/cosmos-block-example](https://github.com/figment-networks/example-subgraph/tree/feature/cosmos-block-example) from [cosmos-block-example](https://github.com/figment-networks/example-subgraph/tree/cosmos-block-example)

Shouldn't this be the other way around?

File: package.json [hunk 6]
```diff
@@ -1,26 +1,25 @@
 {
-  "name": "example",
+  "name": "cosmos-block-example",
   "version": "0.1.0",
-  "repository": "https://github.com/graphprotocol/example-subgraph",
+  "repository": "https://github.com/figment-networks/example-subgraph",
   "license": "MIT",
   "scripts": {
-    "build-contract": "solc contracts/Gravity.sol --abi -o abis --overwrite && solc contracts/Gravity.sol --bin -o bin --overwrite",
     "create": "graph create example --node https://api.thegraph.com/deploy/",
```
Comment (pienkowb) [inline package.json:7]: `example` => `cosmos-block-example`

File: package.json [hunk 7]
```diff
@@ -1,26 +1,25 @@
...
-    "create-local": "graph create example --node http://127.0.0.1:8020",
+    "create-local": "graph create cosmos-example --node http://localhost:8020",
     "codegen": "graph codegen",
     "build": "graph build",
     "deploy": "graph deploy example --ipfs https://api.thegraph.com/ipfs/ --node https://api.thegraph.com/deploy/",
```
Comment (pienkowb) [inline package.json:11]: `example` => `cosmos-block-example`

**Review by pienkowb — APPROVED** [review]

**Pull request merged**